app/src/main/java/com/teamagam/gimelgimel/app/map/actions/send/dynamicLayers/EditDynamicLayerViewModel.java

```diff
@@ -167,15 +203,52 @@ public void onNewTabSelection(int newTabResource) {
     }
   }
 
+  public void onEditDescriptionFabClicked() {
```

Comment:
Clean the mess of this method NESHAMA!

Reply:
And consider moving this job to the navigator or some other place where dialogs like this are managed...